src/schema/bookmarks.ts

```diff
@@ -343,6 +343,7 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
     ): Promise<GQLBookmark[]> => {
       const lastAddedBookmark = await ctx.con.getRepository(Bookmark).findOne({
         where: { userId: ctx.userId },
+        order: { updatedAt: 'DESC', createdAt: 'DESC' },
```

Comment:
why createdAt, updatedAt should be enough?

Reply:
Just to make sure everything worked as intended even with old bookmark but probably this is something that comes out of the box, removing it ✔️